src/libraries/System.Runtime.Extensions/tests/System/Environment.MachineName.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Tests
     public class Environment_MachineName
     {
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Browser)] // throws EntryPointNotFoundException : SystemNative_GetNodeName
```

Comment:
Should this rather throw PlatformNotSupportedException? It would be nice to have a tests that verifies the exception it throws.

Reply:
It shouldn't.  In 3.2, we return "emscripten", which is what `gethostname` returns.  

I will have a follow up PR that addresses this.

Reply:
Is there issue to link to?

Reply:
No, but I'll have a PR right after this one lands.

Reply:
I just added my changes to this PR instead.  We felt it made more sense.

Reply:
Should we update this comment?

Reply:
Updated to not skip the test, instead modified `GetComputerName` to return emscripten on Browser.